.github/workflows/ci-build.yaml

```diff
@@ -151,7 +151,7 @@ jobs:
       - run: make start PROFILE=${{matrix.profile}} AUTH_MODE=client STATIC_FILES=false LOG_LEVEL=info API=${{matrix.test == 'test-api' || matrix.test == 'test-cli' || matrix.test == 'test-java-sdk' || matrix.test == 'test-python-sdk'}} UI=false > /tmp/argo.log 2>&1 &
       - run: make wait
         timeout-minutes: 4
-      - run: make ${{matrix.test}} E2E_TIMEOUT=1m STATIC_FILES=false
+      - run: make ${{matrix.test}} E2E_TIMEOUT=15m STATIC_FILES=false
```

Comment:
I think the E2E_TIMEOUT here is for each workflow or each WaitForWorkflow,
We shouldn't have a workflow that takes up to 15min to run

Reply:
@tczhao  This configuration was never applied into make file. This PR is fixing it. Makefile had 15m time out for entire tests to complete.

Reply:
@sarabala1979 I see what you meant,
I think the original intention of this `E2E_TIMEOUT` is timeout for individual workflows, it is getting loaded here
https://github.com/argoproj/argo-workflows/blob/master/test/e2e/fixtures/e2e_suite.go#L39
https://github.com/argoproj/argo-workflows/blob/master/test/e2e/fixtures/when.go#L245

maybe we should separate these parameters since we have parameters for overall timeout and individual workflow timeout from this PR

Reply:
In this case, `timeout` for waiting on workflows is defaulting to E2E_TIMEOUT correct? And we want to make it so E2E_TIMEOUT only corresponds to overall timeout, not a single workflow?

Reply:
@tczhao you are right. @dpadhiar we can add new variable E2E_SUIT_TIMEOUT This is mainly for entire suite timeout